neat/experiments/UCI/kwargs.py

```diff
@@ -19,11 +19,12 @@
 
         'FITNESS_THRESHOLD' : float("inf"),
 
+        'USE_FITNESS_COEFFICIENT' : False,
         'INITIAL_FITNESS_COEFFICIENT' : 0.1,
         'FINAL_FITNESS_COEFFICIENT' : 0.9,
 
         'POPULATION_SIZE' : 15,
-        'NUMBER_OF_GENERATIONS' : 100,
+        'NUMBER_OF_GENERATIONS' : 20,
```

Comment:
Can you switch this back to 100? I think we decided 100 was a good number for # of generations